src/Nest/CommonAbstractions/Infer/Policy/Policy.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Globalization;
+using Elasticsearch.Net;
+
+namespace Nest
+{
+	public class PolicyId : IUrlParameter, IEquatable<PolicyId>
```

Comment:
Can we use `Id` for this?

Reply:
Sadly, Id allows `long` and this is constrained to `string`. We have a ticket open for consolidating these custom types, maybe look at this then? I imagine we might well end up with `Id` and `IdString` or something similar.

Reply:
Will address in https://github.com/elastic/elasticsearch-net/issues/3672